src/material/snack-bar/simple-snack-bar.ts

```diff
@@ -6,11 +6,20 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {Component, ViewEncapsulation, Inject, ChangeDetectionStrategy} from '@angular/core';
-import {MatSnackBarRef} from './snack-bar-ref';
+import {ChangeDetectionStrategy, Component, Inject, ViewEncapsulation} from '@angular/core';
 import {MAT_SNACK_BAR_DATA} from './snack-bar-config';
+import {MatSnackBarRef} from './snack-bar-ref';
 
 
+/**
+ * Internal interface for a simple snack bar component..
+ * @docs-private
+ */
+export interface BaseSimpleSnackBar {
```

Comment:
How about something like `TextOnlySnackbar` or `MessageOnlySnackbar`

Reply:
Works for me, I think `BasicSnackBar` could also work, but it's `docs-private` anyways and I'm not too worried about it. Went with `TextOnlySnackBar`